.gitignore

```diff
@@ -24,3 +24,5 @@ bin
 *~
 
 hack/*.kubeconfig
+
+alice-oil.kubeconfig
```

Comment:
hum, why `alice-oil.kubeconfig` here? It should be generated inside the `hack` folder, and so ignored.